### ISSUE TITLE
`--inline-snapshot=fix/create` shouldn't cause the review diff to be printed

### ISSUE THREAD

**Issue by samuelcolvin** [15r10nk/inline-snapshot#128]
As per #123, my bad for creating an issue with multiple points.

Currently when I use `--inline-snapshot=fix` or `--inline-snapshot=create` I get a massive diff printed. This is the result of a mixture of ruff formatting differing from this library, my preferences and `IsNow()` replacing datetimes.

As per #123 `Is()` might solve some of these, but not all.

I'd like (either as the default, or via a config argument) to be able to disable the diff being printed.

Another downside of the big diff is that it means I can't set `export INLINE_SNAPSHOT_DEFAULT_FLAGS=create` by default.

**Comment by alexmojaki** [comment]
`--inline-snapshot=fix,short-report` works.

When the report is printed, it says underneath:

```
These changes are not applied.
Use --inline-snapshot=update to apply them, or use the interactive mode with --inline-snapshot=review
```

It should also suggest `short-report` in that message, because I only found this in the context of this discussion.

**Comment by 15r10nk** [comment]
#129 will fix this issue

`--inline-snapshot=fix` will only report what it has fixed and nothing more.
`--inline-snapshot=fix,report` will also report other changes which is equivalent to the current behaviour.

**Comment by samuelcolvin** [comment]
🎉 thanks so much!

**Comment by 15r10nk** [comment]
Fixed in 0.14.0. I will work on how inline-snapshot generates code. This huge diffs shouldn't be there in the first place.
